packages/core/modules/opus_measurement.py

```diff
@@ -265,53 +246,35 @@ def start_opus(self) -> None:
         except AttributeError:
             pass
 
-    def opus_application_running(self) -> bool:
-        """Checks if OPUS is already running by identifying the window.
+    def opus_is_running(self) -> bool:
+        """Checks if OPUS is already running by searching for processes with
+        the executable `opus.exe` or `OpusCore.exe`
 
-        Returns:
-        False if Application is currently not running on OS
-        True if Application is currently running on OS
-        """
-        assert sys.platform == "win32"
+        Returns: `True` if Application is currently running and `False` if not."""
 
-        # FindWindow(className, windowName)
-        # className: String, The window class name to find, else None
-        # windowName: String, The window name (ie,title) to find, else None
-        opus_username = self.config["opus"]["username"]
-        opus_windows_name = (
-            f"OPUS - Operator: {opus_username}  (Administrator) - [Display - default.ows]"
-        )
-        try:
-            if _win32ui.FindWindow(
-                None,
-                opus_windows_name,
-            ):
+        for p in psutil.process_iter():
+            if p.name() in ["opus.exe", "OpusCore.exe"]:
                 return True
-            return False
-        except _win32ui.error:
-            return False
+
+        return False
```

Comment:
Important change. This should prevent multiple OPUS windows to open, when the windows title has a mismatch